vagrant/sync-usr-share.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# This file is part of openmediavault.
+#
+# @license   http://www.gnu.org/licenses/gpl.html GPL Version 3
+# @author    Volker Theile <volker.theile@openmediavault.org>
+# @copyright Copyright (c) 2009-2023 Volker Theile
+#
+# openmediavault is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# any later version.
+#
+# openmediavault is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with openmediavault. If not, see <http://www.gnu.org/licenses/>.
+
+set -e
+
+rsync -av -m /home/vagrant/openmediavault/openmediavault/usr/share/ /usr/share/
```

Comment:
This syncs from vagrant share to /usr/share/openmediavault
I used rsync as then it shows me what has changed, to make sure it's expected